src/index.html

```diff
@@ -17,6 +30,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo Moyo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_menu">
+          <li>
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+              data-qa="hover"
+            >
+              Laptops &amp; computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
```

Comment:
You should add empty lines between multiline sibling blocks of HTML to add some "air" and simplify reading. But don't add them between parent and child elements.